manifest.aspnet.json

```diff
@@ -0,0 +1,186 @@
+{
+  "readmePath": "README.aspnet.md",
```

Comment:
This readmePath property should be completely removed.  The aspnet readme comes from line 11.